kani-driver/src/session.rs

```diff
@@ -191,6 +203,19 @@ impl KaniSession {
 
         Ok(Some(process))
     }
+
+    pub fn with_timer<T, F>(&self, func: F, description: &str) -> T
```

Comment:
Might be worth creating a "common" or "utils" crate at some point that includes this function (and similar ones) to avoid duplication.

Reply:
I totally agree. Not sure it's worth it in this PR though